src/storage/BaseProcessor.h

```diff
@@ -101,6 +101,9 @@ class BaseProcessor {
 
   void handleAsync(GraphSpaceID spaceId, PartitionID partId, nebula::cpp2::ErrorCode code);
 
+  nebula::cpp2::ErrorCode checkStatType(const meta::SchemaProviderIf::Field* field,
```

Comment:
an input parameter should be const references.
please consult google style https://google.github.io/styleguide/cppguide.html#Inputs_and_Outputs